src/Razor/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/src/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>ASP.NET Core design time hosting infrastructure for the Razor view engine.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>$(DefaultNetFxTargetFramework);netstandard2.0</TargetFrameworks>
```

Comment:
Changes w/in the Razor SDK probably aren't useful. @NTaylorMullen❔

Reply:
Yup. These extensions are packaged in the SDK. Unless we are shipping the netfx versions for some reason, I'd avoid this change